roles/testnode/tasks/apt_systems.yml

```diff
@@ -3,7 +3,7 @@
 # For anything ubuntu < 15.04 or debian <=7 we still do.
 - name: Setup local repo files.
   include: apt/repos.yml
-  when: ansible_distribution_major_version|int < 15
+  when: ansible_pkg_mgr == "apt" and ansible_distribution_major_version|int < 15
```

Comment:
http://docs.ansible.com/ansible/playbooks_conditionals.html#applying-when-to-roles-and-includes seems to imply that it might evaluate them all, but they should all get the same 'when' conditional, no?